docs/config/index.md

```diff
@@ -255,6 +257,12 @@ export default ({ command, mode }) => {
           changeOrigin: true,
           rewrite: (path) => path.replace(/^\/api/, '')
         }
+        // with RegEx
+        '^\/fallback\/.*': {
```

Comment:
The slash doesn't need to be escaped here. Only backslash itself needs escaping.